src/librustc_typeck/check/callee.rs

```diff
@@ -29,7 +29,7 @@ use rustc::hir;
 pub fn check_legal_trait_for_method_call(ccx: &CrateCtxt, span: Span, trait_id: DefId) {
     if ccx.tcx.lang_items.drop_trait() == Some(trait_id) {
         struct_span_err!(ccx.tcx.sess, span, E0040, "explicit use of destructor method")
-            .span_label(span, &format!("call to destructor method"))
+            .span_label(span, &format!("explicit destructor calls not allowed"))
```

Comment:
That's true sometimes. We don't yet have the rules set in stone, rather more of a best effort. 

With some errors, like this one, we use a rephrasing. Mostly this is because we're not sure where people will look first: the title or the label.